src/utils/encoding/mod.rs

```diff
@@ -79,15 +83,123 @@ pub fn prover_id_from_u64(id: u64) -> ProverId {
     prover_id
 }
 
+// Unsigned VarInt (Uvi) Bytes
+pub struct UviBytes {
```

Comment:
I'd put it into a different file but it's not a big deal. In general, I like when mod is just a collection of exports.

Reply:
Moved.